src/includes/getting-started-install/javascript.mdx

```diff
@@ -10,3 +10,9 @@ npm install --save @sentry/browser @sentry/tracing
 We also support alternate [installation methods](/platforms/javascript/install/).
 
 </Note>
+
+<Note>
+
+Are you updating your Sentry SDK to the latest version? Check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn more about breaking changes!
+
+</Note>
```

Comment:
Suggest changing the language here so we can take this out of a note and make it body text. Right now, it comes right after another note that's already on the page, and having two notes in a row with no body text between them is something we should avoid. Also, we don't use exclamation points in docs.

```suggestion
 If you're updating your Sentry SDK to the latest version, check out our [migration guide](https://github.com/getsentry/sentry-javascript/blob/7.x/MIGRATION.md) to learn more about breaking changes.

```